### ISSUE TITLE
Peer Review: Milestone 3

### ISSUE THREAD

**Issue by aralshi** [airavata-courses/Gateway-Falcons#73]
Hi team,

I was able to test the application using frontend and also could check the kubernetes deployment using the dashboard. Documentation in Wiki was really helpful to understand tasks achieved and how I can validate them. Front end is user friendly and clean. It also easy to check the micro-service's branches using wiki. 

Few Suggestions:
1. Add revised architecture diagram to wiki to explain Kubernetes setup and how is your application interacting within the cluster.
2. Jenkins file used instead to script in the Jenkins job. That way anyone can replicate the application without any assistance.
3. Media and donate links open website outside the application, so the link should be opened on new tab of the browser rather that loading in the same tab. This way you guys won’t loose the viewers.

Also few questions:
1. All Backend services I checked are in python or node js. I did miss the third technology service?
2. Where are the cofig.ini files?
3. Is it a single biker application as there is no option to select the biker to view information of?
4. How are you differentiating biker from viewer as there is no register/login functionality?

Overall great job!!

**Comment by Chirag-Galani** [comment]
Hi Ankita,
Thanks for your feedback. We will definitely add/alter feature after discussing them with our client.
Also, for your questions:

**All Backend services I checked are in python or node js. I did miss the third technology service?**
Yeah. We talked to the professor and did convert the Java Microservice to NodeJS for smoother deployment and to prevent some SpringBoot CORS issues.

**Where are the cofig.ini files?**
The config.ini files are on the jenkins and other development servers. Also, they are added inside the docker containers to prevent leaking of credentials

**Is it a single biker application as there is no option to select the biker to view information of?**
As of now, we are developing the application for a single biker and have added all the features, functionalities to cover different data points for a single biker. We plan to make it a generalist biker/sportsperson solution in the next milestone (maybe)

**How are you differentiating biker from viewer as there is no register/login functionality?**
The biker doesn't manually upload the data. The website is only for viewers (fans, friends and family) who want to see and motivate our client on his ride from east to the west coast. He has linked his account to different services like Fitbit, Strava, Wahoo and MyFitnessPal. Our code simply fetches the data on a periodic basis and stores it on our database.
